cloudflared/rootfs/etc/s6-overlay/scripts/10-cloudflared-config.sh

```diff
@@ -7,6 +7,58 @@
 # given hostname(s)
 # ==============================================================================
 
+# ------------------------------------------------------------------------------
+# Checks for legacy options and removes them
+# ------------------------------------------------------------------------------
+checkLegacyOptions() {
```

Comment:
@elcajon I would love to hear your thoughts on checking this here, including keeping the migration from the 'data_folder'. The alternative would be just not to care about the legacy options being present in the config and not start if the config does not meet our "normal" checks

Reply:
I would say that this option is more user friendly then just failing with an error. Let's start with this for some minor or major releases before removing the functionality again.